Dockerfile.windows

```diff
@@ -171,70 +171,28 @@ ENV GO_VERSION=1.7.3 `
     FROM_DOCKERFILE=1
 
 RUN `
-  $ErrorActionPreference = 'Stop'; `
-  $ProgressPreference = 'SilentlyContinue'; `
-  `
-  Function Test-Nano() { `
-    $EditionId = (Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion' -Name 'EditionID').EditionId; `
-    return (($EditionId -eq 'ServerStandardNano') -or ($EditionId -eq 'ServerDataCenterNano') -or ($EditionId -eq 'NanoServer')); `
-  }`
-  `
-  Function Download-File([string] $source, [string] $target) { `
```

Comment:
@jhowardmsft I simplified Dockerfile based on Nanoserver too. It works great (although subsequent builds don't work becuase of the the depedency on the assorted resource/message pre-processors). Are you saying that there's a later build of Nanoserver that has:

1. PowerShell
2. .NET framework installed with `System.IO.FileStream`, `System.Net.Http` et.al. available
3. Has had `Invoke-WebRequest` surgically removed, breaking everyone that has incorporated `Invoke-WebRequest` in PowerShell scripts written to run on Nanoserver

?